pkg/network/http/model.go

```diff
@@ -40,6 +40,22 @@ func (k *httpBatchKey) Prepare(n httpNotification) {
 	k.page_num = C.uint(int(n.batch_idx) % HTTPBatchPages)
 }
 
+func (tx *httpTX) PostProcess() {
+	postProcessFragment(tx)
+}
+
+func (tx *httpTX) fragmentSize() int {
+	return int(tx.fragment_sz)
+}
+
+func postProcessFragment(tx *httpTX) {
+	b := *(*[HTTPBufferSize]byte)(unsafe.Pointer(&tx.request_fragment))
+	fSz := tx.fragmentSize()
+	for i := fSz; i < HTTPBufferSize; i++ {
+		b[i] = 0
+	}
```

Comment:
I wonder if having a global var `zero [HTTPBufferSize]byte` and using `copy(b[fSz:], zero)` would be faster? Maybe worth benchmarking?